pull_request_template.md

```diff
@@ -1,13 +1,4 @@
 ### What this PR does (required):
-- list PR functionality here
+The request Controller is created. This is where the logic and manipulation to create a request is done.
```

Comment:
We shouldn't modify the template. Instead fill out the template on GitHub's interface when preparing a PR 👍🏼 (check out step 5 [here](https://github.com/hatchways/team-snowman/pull/62#discussion_r755975724))